src/test/java/ticTacToe/game/MarkTest.java

```diff
@@ -0,0 +1,37 @@
+package ticTacToe.game;
+
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertTrue;
```

Comment:
IntelliJ doesn't make it obvious enough, but prefer the `org.junit.*` imports over `junit.framework.*` as the `org.junit` imports are the ones from the newer version.